packages/navigator/src/browser/open-editors-widget/navigator-deleted-editor-decorator.ts

```diff
@@ -0,0 +1,77 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject, postConstruct } from '@theia/core/shared/inversify';
+import { DepthFirstTreeIterator, NavigatableWidget, Tree, TreeDecoration, TreeDecorator } from '@theia/core/lib/browser';
+import { FileSystemFrontendContribution } from '@theia/filesystem/lib/browser/filesystem-frontend-contribution';
+import { Emitter } from '@theia/core';
+import { OpenEditorNode } from './navigator-open-editors-tree-model';
+import { FileChangeType } from '@theia/filesystem/lib/browser';
+
+@injectable()
+export class NavigatorDeletedEditorDecorator implements TreeDecorator {
+
+    @inject(FileSystemFrontendContribution)
+    protected readonly fileSystemContribution: FileSystemFrontendContribution;
+
+    readonly id = 'theia-deleted-editor-decorator';
+    protected readonly decorationsChangedEmitter = new Emitter();
+    readonly onDidChangeDecorations = this.decorationsChangedEmitter.event;
```

Comment:
minor - related to a previous comment:

```suggestion
    protected readonly onDidChangeDecorationsEmitter = new Emitter();
    readonly onDidChangeDecorations = this.decorationsChangedEmitter.event;
```

Reply:
👍